src/game/default/g_util.c

```diff
@@ -547,6 +551,24 @@ g_team_t *G_OtherTeam(g_team_t *t) {
 }
 
 /*
+ *	Return the number of players on the given team.
+ */
+size_t G_TeamSize(g_team_t *team) {
+	g_client_t *cl;
+	size_t count = 0;
+	
+	for (int32_t i = 0; i < sv_max_clients->integer; i++){
+		if (!g_game.entities[i + 1].in_use)
+			continue;
+			
+		cl = g_game.entities[i + 1].client;
```

Comment:
No, the stack space for the variable is allocated in function scope, either way. The difference is that, by reducing the variable's AST scope, the code is easier to maintain and follow. Having to declare variables at the beginning of a function is an old C89 requirement, which is why Quake2's original code did this. But reducing variable scope and using const are both best practices for modern c development.